list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code lacks test coverage for the list functionality.

Recommendations:
- **Unit Tests**: Implement unit tests for the list functionality to ensure it handles various scenarios, such as empty databases and database errors.
- **Example**:
  ```php
  public function testListBooks() {
      $mockConnection = $this->createMock(mysqli::class);
      $repository = new BookRepository($mockConnection);
      $books = $repository->listBooks();
      $this->assertIsArray($books);
  }
  ```